sdk/chaos/arm-chaos/src/chaosManagementClient.ts

```diff
@@ -1,183 +1,80 @@
-/*
- * Copyright (c) Microsoft Corporation.
- * Licensed under the MIT License.
- *
- * Code generated by Microsoft (R) AutoRest Code Generator.
- * Changes may cause incorrect behavior and will be lost if the code is regenerated.
- */
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
 
-import * as coreClient from "@azure/core-client";
-import * as coreRestPipeline from "@azure/core-rest-pipeline";
 import {
-  PipelineRequest,
-  PipelineResponse,
-  SendRequest,
-} from "@azure/core-rest-pipeline";
-import * as coreAuth from "@azure/core-auth";
+  createChaosManagement,
+  ChaosManagementContext,
+  ChaosManagementClientOptionalParams,
+} from "./api/index.js";
 import {
-  CapabilitiesImpl,
-  CapabilityTypesImpl,
-  ExperimentsImpl,
-  OperationStatusesImpl,
-  OperationsImpl,
-  TargetTypesImpl,
-  TargetsImpl,
-} from "./operations/index.js";
+  OperationStatusesOperations,
+  _getOperationStatusesOperations,
+} from "./classic/operationStatuses/index.js";
+import { TargetTypesOperations, _getTargetTypesOperations } from "./classic/targetTypes/index.js";
 import {
-  Capabilities,
-  CapabilityTypes,
-  Experiments,
-  OperationStatuses,
-  Operations,
-  TargetTypes,
-  Targets,
-} from "./operationsInterfaces/index.js";
-import { ChaosManagementClientOptionalParams } from "./models/index.js";
+  ExperimentExecutionsOperations,
+  _getExperimentExecutionsOperations,
+} from "./classic/experimentExecutions/index.js";
+import { ExperimentsOperations, _getExperimentsOperations } from "./classic/experiments/index.js";
+import {
+  CapabilityTypesOperations,
+  _getCapabilityTypesOperations,
+} from "./classic/capabilityTypes/index.js";
+import { TargetsOperations, _getTargetsOperations } from "./classic/targets/index.js";
+import { OperationsOperations, _getOperationsOperations } from "./classic/operations/index.js";
+import {
+  CapabilitiesOperations,
+  _getCapabilitiesOperations,
+} from "./classic/capabilities/index.js";
+import { Pipeline } from "@azure/core-rest-pipeline";
+import { TokenCredential } from "@azure/core-auth";
 
-export class ChaosManagementClient extends coreClient.ServiceClient {
-  $host: string;
-  apiVersion: string;
-  subscriptionId?: string;
+export { ChaosManagementClientOptionalParams } from "./api/chaosManagementContext.js";
 
-  /**
-   * Initializes a new instance of the ChaosManagementClient class.
-   * @param credentials Subscription credentials which uniquely identify client subscription.
-   * @param subscriptionId GUID that represents an Azure subscription ID.
-   * @param options The parameter options
-   */
+export class ChaosManagementClient {
+  private _client: ChaosManagementContext;
+  /** The pipeline used by this client to make requests */
+  public readonly pipeline: Pipeline;
+
+  /** Chaos Management Client */
   constructor(
-    credentials: coreAuth.TokenCredential,
+    credential: TokenCredential,
```

Comment:
for modular - are all operation groups with subid param?
for hlc - why we have constructor overload?